components/OlasTokenPage/ActualEmissionsChart.jsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Chart,
+  Filler,
+  LineElement,
+  LinearScale,
+  PointElement,
+  Tooltip,
+} from 'chart.js';
+import {
+  EMISSIONS_CHART_COLORS,
+  getEmissionsChartOptions,
+} from 'common-util/charts';
+import PropTypes from 'prop-types';
+import { Line } from 'react-chartjs-2';
+import { LegendItem } from './LegendItem';
+import { emissionType } from './types';
+
+Chart.register(LineElement, LinearScale, PointElement, Filler, Tooltip);
+
+export const ActualEmissionsChart = ({ emissions, loading }) => {
+  const maxAvailableEmissions = emissions.map(
+    (item) =>
+      parseFloat(item.availableDevIncentives || 0) +
```

Comment:
better to avoid parseFloat and use BigInt to maintain precision, e.g. `BigInt(item.availableDevIncentives || 0) + ...`

Reply:
This returns a type error as the Line component does not take bigint values

Reply:
Done now :)